packages/pds/src/api/com/atproto/admin/getSubjectState.ts

```diff
@@ -0,0 +1,45 @@
+import { CID } from 'multiformats/cid'
+import { AtUri } from '@atproto/syntax'
+import { Server } from '../../../../lexicon'
+import AppContext from '../../../../context'
+import { OutputSchema } from '../../../../lexicon/types/com/atproto/admin/getSubjectState'
+import { AuthRequiredError, InvalidRequestError } from '@atproto/xrpc-server'
+
+export default function (server: Server, ctx: AppContext) {
+  server.com.atproto.admin.getSubjectState({
+    auth: ctx.roleVerifier,
+    handler: async ({ params, auth }) => {
+      const access = auth.credentials
+      // if less than admin access then cannot perform a takedown
+      if (!access.moderator) {
+        throw new AuthRequiredError(
+          'Must be a full moderator to update subject state',
+        )
+      }
```

Comment:
There's no update happening here, so I think we can safely skip this check.

Reply:
ah yup good call 👍